app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
+        raise ValueError("You line should start with cp")
+    if len(command) != 3:
+        raise ValueError("Message should be: cp <source_file> <destination_file>")
+    if command[1] == command[2]:
+        raise ValueError("There should be two different names of files")
+    elif ".txt" in (command[1] and command[2]):
```

Comment:
The '.txt' check is not correct. The current check will return True if '.txt' is in either command[1] or command[2], not both. You need to check both individually. Also, it would be better to check if the file names end with '.txt', not just if they contain '.txt'.